### ISSUE TITLE
Rename "post titles" to "headlines" in UI

### ISSUE THREAD

**Issue by rmccue** [humanmade/experiments#40]
To avoid the jargon, we should use a more generic term for the experiment.

**Comment by roborourke** [comment]
Headline is still jargon but for publishing isn't it? Also doesn't really disambiguate headers in the content vs the post title. Agree with the premise here in theory but how do we know this is a problem that needs solving?

**Comment by rmccue** [comment]
Yes and no. "Post" is confusing when you're not actually on a post-type CPT at least, so we should change it to just "title" at least.

"Headline" is jargon, but in the same realm as "title" IMO. Either is OK, it just struck me how weird it was to see "post titles" while on a page.

**Comment by roborourke** [comment]
Yeah that's totally fair. Would be easy to use the current post type label instead
